run.sh

```diff
@@ -1,18 +1,19 @@
 #!/bin/bash
-echo "Running in $PWD" 
 previous_gazebo_model_path=${GAZEBO_MODEL_PATH}
 previous_gazebo_plugin_path=${GAZEBO_PLUGIN_PATH}
-export SWARMATHON_APP_ROOT="$PWD"
-export GAZEBO_MODEL_PATH="$PWD/simulation/models"
-export GAZEBO_PLUGIN_PATH="$PWD/build/gazebo_plugins"
-source "$PWD/devel/setup.bash"
+export SWARMATHON_APP_ROOT="$(catkin locate)"
```

Comment:
ditto
```suggestion
SWARMATHON_APP_ROOT="$(catkin locate)"
```

Reply:
@Carter90 awesome that you're working on this!